Eastern-Research-Group/mywaterway
# feature/20_edits-to-NARS-data

## Related Issues:
* https://jira.epa.gov/browse/HMW-20

## Main Changes:
* Updated the national page text and made the national page lookup file more flexible.

## Steps To Test:
1. Navigate to http://localhost:3000/national
2. Verify everything loads in
3. Verify the text matches up with the following document: [Final NARS edits Hows My Water Way Revisions_Clean.docx](https://github.com/Eastern-Research-Group/mywaterway/files/7978834/Final.NARS.edits.Hows.My.Water.Way.Revisions_Clean.docx)

## TODO:
- [ ] I would like to move the images to S3 as well. Unfortunately that requires changes to the server side code, so we have to hold off until the next full release. I created the [HMW-121](https://jira.epa.gov/browse/HMW-121) ticket to do this.
